medusa/process_tv.py

```diff
@@ -453,26 +457,17 @@ def unrar(self, path, rar_files, force=False):
                             break
 
                     if not skip_extraction:
-                        rar_handle.extract(path=path, withSubpath=False, overwrite=False)
+                        rar_handle.extract(path)
```

Comment:
from what @duramato said some time ago, the RARed torrents are only for the video file. 
Sample, NFO and everything else is not rared